lib/editor/components/ScheduleExceptionForm.js

```diff
@@ -87,12 +90,23 @@ export default class ScheduleExceptionForm extends Component<Props> {
     })
   }
 
+  _onCustomScheduleChange = (evt: SyntheticInputEvent<HTMLInputElement>) => {
+    const {activeComponent, activeEntity, updateActiveGtfsEntity} = this.props
+    updateActiveGtfsEntity({
+      component: activeComponent,
+      entity: activeEntity,
+      props: {custom_schedule: evt.target.value}
+    })
+  }
+
   _renderExceptionExemplars (exemplar: string, value: number): string {
     switch (value) {
       case EXCEPTION_EXEMPLARS.SWAP:
-        return 'Swap, add, or remove'
+        return this.messages('swapAddOrRemove')
       case EXCEPTION_EXEMPLARS.NO_SERVICE:
-        return toSentenceCase(exemplar.replace('_', ' '))
```

Comment:
Are we still using the exemplar?

Reply:
We're just using an i18n string now so it's separate from the exemplar variable name now. This is better bc now it can be translated.